core/src/main/java/com/nisovin/magicspells/spells/targeted/SpawnEntitySpell.java

```diff
@@ -488,6 +491,7 @@ private class AttackMonitor implements Listener {
 		private final LivingEntity monster;
 		private final String[] args;
 		private final float power;
+		private final boolean cancelAttack;
```

Comment:
Not necessary to have the option as a field here; just reference the option from the parent spell.